### PR TITLE
[EXPLORER] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Comment by AndreiMiloiu** [comment]
I can't find the reason why the test fails.

File: base/shell/explorer/lang/ro-RO.rc [hunk 1]
```diff
@@ -108,112 +108,112 @@ BEGIN
...
-    DEFPUSHBUTTON    "Ign&oră", IDIGNORE, 215, 60, 50, 14
-    PUSHBUTTON       "&Redenumește", IDC_FILE_RENAME, 159, 60, 50, 14
+    DEFPUSHBUTTON    "&Ignorare", IDIGNORE, 215, 60, 50, 14
+    PUSHBUTTON       "&Redenumire", IDC_FILE_RENAME, 159, 60, 50, 14
     LTEXT            "", IDC_STATIC, 7, 51, 258, 1, NOT WS_GROUP
-    AUTOCHECKBOX     "N&u efectua această verificare la pornire", IDC_NOTCHECK, 7, 62, 139, 10
+    AUTOCHECKBOX     "&Nu se efectuează această verificare la pornire", IDC_NOTCHECK, 7, 62, 139, 10
     ICON             32515, IDC_STATIC, 7, 7, 21, 20
-    LTEXT            "În calculator există un fișier sau folder numit %s care poate cauza probleme în funcționarea corectă a anumitor aplicații. Redenumirea sa în %s ar rezolva această problemă. Doriți să-l redenumiți acum?", IDC_STATIC, 34, 7, 224, 37
+    LTEXT            "În computer există un fişier sau un folder denumit "%s" care împiedică funcţionarea corectă a unor aplicaţii. Problema se va soluţiona dacă el este redenumit "%s". Îl redenumiţi acum?", IDC_STATIC, 34, 7, 224, 37
```
Comment (binarymaster) [inline base/shell/explorer/lang/ro-RO.rc:162]:
```suggestion
    LTEXT            "În computer există un fişier sau un folder denumit ""%s"" care împiedică funcţionarea corectă a unor aplicaţii. Problema se va soluţiona dacă el este redenumit ""%s"". Îl redenumiţi acum?", IDC_STATIC, 34, 7, 224, 37
```

**Comment by binarymaster** [comment]
> I can't find the reason why the test fails.

Escaping the double quote properly...
```
/home/runner/work/reactos/reactos/src/base/shell/explorer/lang/ro-RO.rc:162: syntax error
```
Fixed.

**Comment by AndreiMiloiu** [comment]
Thank you @binarymaster!

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**